src/lib/getConfig.ts

```diff
@@ -1,14 +1,22 @@
 import chalk from 'chalk'
 import fs from 'fs-extra'
 import path from 'path'
+import yaml from 'yaml'
 import { PortmanConfig } from '../types'
 
 export const getConfig = async (configPath: string | undefined): Promise<PortmanConfig> => {
   let config = {}
 
   if (configPath && (await fs.pathExists(path.resolve(configPath)))) {
-    config = await import(path.resolve(configPath)).then(module => module.default)
-  } else {
+    // Check if config is YAML file
+    if (configPath.indexOf('.yaml') >= 0 || configPath.indexOf('.yml') >= 0) {
```

Comment:
`configPath.includes('.yaml') || configPath.includes('.yml')`

Reply:
corrected and resolved